tools/ci.sh

```diff
@@ -220,7 +220,7 @@ function ci_esp32_build_cmod_spiram_s2 {
     ci_esp32_build_common
 
     make ${MAKEOPTS} -C ports/esp32 \
-        USER_C_MODULES=../../../examples/usercmodule/micropython.cmake \
+        USER_C_MODULES=../../examples/usercmodule/micropython.cmake \
```

Comment:
It's frustrating that this will break anyone who has an esp32 project using `USER_C_MODULES` and relative paths. However, at the same time this is a better way to express the relative path and it's how all the other ports work -so I think better to do it.